src/Lib.hs

```diff
@@ -6,17 +6,17 @@ import           Network.Wai.Handler.Warp
 import           Routes
 import           Servant
 
-startApp :: IO ()
-startApp = run 8081 app
+startApp :: (MonadIO m) => m ()
```

Comment:
By adding a `m` parameter to the `AppM` type declared bellow, we can now abstract here a bit more using MonadIO instance of IO directly. Honestly, I don't really know how useful that is at this function level, but hey.